tarodb/assets_store.go

```diff
@@ -762,6 +788,61 @@ func (a *AssetStore) FetchAllAssets(ctx context.Context,
 	return dbAssetsToChainAssets(dbAssets, assetWitnesses)
 }
 
+// FetchManagedUTXOs fetches all UTXOs we manage.
+func (a *AssetStore) FetchManagedUTXOs(ctx context.Context) (
+	[]*ManagedUTXO, error) {
+
+	var (
+		utxos []ManagedUTXORow
+		err   error
+	)
+
+	readOpts := NewAssetStoreReadTx()
+	dbErr := a.db.ExecTx(ctx, &readOpts, func(q ActiveAssetsStore) error {
+		utxos, err = q.FetchManagedUTXOs(ctx)
+		return err
+	})
+	if dbErr != nil {
+		return nil, dbErr
+	}
+
+	managedUtxos := make([]*ManagedUTXO, len(utxos))
+	for i, u := range utxos {
```

Comment:
nit: seems to me that `i` is unused?

Reply:
Used to index into `managedUtxos` on line 826 😄